packages/ui/src/components/DatePicker/SDatePickerPanelTime.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+interface Props {
+  value: string
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  value: '00:00',
+})
+
+const emit = defineEmits(['updateTime'])
+
+let timeArray = ref<string[]>([])
+
+for (let i = 0; i < 24; i++) {
+  const hour = i < 10 ? `0${i}` : i
+  timeArray.value.push(`${hour}:00`)
+  timeArray.value.push(`${hour}:30`)
+}
```

Comment:
This code will run for each component initialization, but actually it could be run once at all. So, I propose to move it out from this component to `consts.ts`.

Also, change `let timeArray` to `const timeArray`.